### ISSUE TITLE
Failed to retrieve authentication checksums for provider

### ISSUE THREAD

**Issue by TGNThump** [larstobi/terraform-provider-multipass#5]
I get the following error while trying to install the latest two versions via terraform registry
```
╷
│ Error: Failed to install provider
│
│ Error while installing larstobi/multipass v1.2.1: could not query provider registry for registry.terraform.io/larstobi/multipass: failed to retrieve  
│ authentication checksums for provider: 404 Not Found
╵
```

**Comment by larstobi** [comment]
Sorry, I had to remove 1.2.1 since it had was missing the schema upgrade function. Please use 1.2.0 instead, and I will publish 1.2.2 soon with support for output of IPv4 address.

**Comment by larstobi** [comment]
I can see that the Terraform Registry has not been updated. It seems it can't remove releases, so I have published the v1.2.1 release with no change from v1.2.0 instead. Sorry about that. I'm still learning how publishing to the registry works. Anyway, thanks for letting me know. I just checked, and I could download the plugin version now. :)
